.github/workflows/release.yml

```diff
@@ -1,92 +1,81 @@
-    runs-on: pub-hk-ubuntu-22.04-small
-    permissions:
-      contents: write
-      pull-requests: write
-    env:
-      REQUESTED_BUILDPACK_ID: ${{ github.event.inputs.requested_buildpack_id }}
-    steps:
-      - id: checkout
-        name: "Checkout code"
-        uses: actions/checkout@v3
-      - id: setup-pack
-        name: "Setup pack"
-        uses: buildpacks/github-actions/setup-pack@v5.2.0
-      - name: Install yj
-        uses: buildpacks/github-actions/setup-tools@v5.2.0
-      - id: dockerhub-login
-        name: "Login to Docker Hub"
-        uses: docker/login-action@v2
-        with:
-          username: ${{ secrets.DOCKER_HUB_USER }}
-          password: ${{ secrets.DOCKER_HUB_TOKEN }}
-      - id: install-musl-tools
-        run: sudo apt-get install musl-tools
-      - id: install-rust-toolchain
-        name: "Install Rust toolchain"
-        uses: actions-rs/toolchain@v1
-        with:
-          toolchain: stable
-          target: "x86_64-unknown-linux-musl"
-      - id: install-libcnb-cargo
-        name: "Install libcnb-cargo"
-        uses: actions-rs/install@v0.1
-        with:
-          crate: libcnb-cargo
-          version: latest
-      - id: package
-        name: "Package buildpack and publish to container registry"
-        run: ./.github/scripts/release-workflow-package-push.sh
-        shell: bash
-      - id: add-registry-entry
-        name: "Request Registry Entry"
-        uses: docker://ghcr.io/buildpacks/actions/registry/request-add-entry:4.0.0
-        with:
-          token: ${{ secrets.CNB_REGISTRY_RELEASE_BOT_GITHUB_TOKEN }}
-          id: ${{ steps.package.outputs.id }}
-          version: ${{ steps.package.outputs.version }}
-          address: ${{ steps.package.outputs.address }}
-      - id: create_release
-        name: Create Release
-        uses: actions/create-release@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          tag_name: ${{ steps.package.outputs.id }}_${{ steps.package.outputs.version }}
-          release_name: ${{ steps.package.outputs.id }} ${{ steps.package.outputs.version }}
-          body: |
-            Find the changelog here: [CHANGELOG](${{ steps.package.outputs.path }}/CHANGELOG.md)
-          draft: false
-          prerelease: false
-      - id: release_package
-        name: Upload buildpackage to release
-        uses: svenstaro/upload-release-action@v2
-        with:
-          repo_token: ${{ secrets.GITHUB_TOKEN }}
-          file: ${{ steps.package.outputs.package }}
-          tag: ${{ steps.package.outputs.id }}_${{ steps.package.outputs.version }}
-          overwrite: true
-      - id: prepare-pr
-        name: "Prepare PR with version bumps and CHANGELOG updates"
-        run: ./.github/scripts/release-workflow-prepare-pr.sh "${{ steps.package.outputs.id }}" "${{ steps.package.outputs.version }}" "docker://${{ steps.package.outputs.address }}"
```

Comment:
These scripts are now unused and can be deleted:
https://github.com/heroku/buildpacks-nodejs/tree/use_release_automation/.github/scripts

...and after deleting those I think these are also unused:

```
‎‎common/meta-build.sh‎
test/meta-buildpacks/nodejs/build.sh‎
‎test/meta-buildpacks/nodejs-function/build.sh‎
```

And this readme section can be removed:
https://github.com/heroku/buildpacks-nodejs/blob/da26d191d0631ea8fa2a8fe01c2282656b34b841/README.md?plain=1#L44-L55

Reply:
@edmorley the `build.sh` files in the `test/meta-*` projects needs to stick around a bit longer due to this code in [cutlass](https://github.com/heroku/cutlass/blob/main/lib/cutlass/local_buildpack.rb#L93-L116)

Reply:
> the `build.sh` files in the `test/meta-*` projects needs to stick around a bit longer due to this code in [cutlass](https://github.com/heroku/cutlass/blob/main/lib/cutlass/local_buildpack.rb#L93-L116)

Ah yes, true :-)

@colincasey Though I believe this bit of my review comment still applies?

> These scripts are now unused and can be deleted:
> https://github.com/heroku/buildpacks-nodejs/tree/use_release_automation/.github/scripts

Reply:
@edmorley looped back to cover this with #577 and #578